yellowbrick/cluster/silhouette.py

```diff
@@ -115,7 +115,7 @@ class SilhouetteVisualizer(ClusteringScoreVisualizer):
     def __init__(self, estimator, ax=None, colors=None, is_fitted="auto", **kwargs):
 
         # Initialize the visualizer bases
-        super(SilhouetteVisualizer, self).__init__(estimator, ax=ax, **kwargs)
+        super(SilhouetteVisualizer, self).__init__(estimator, ax=ax, is_fitted=is_fitted, **kwargs)
```

Comment:
```suggestion
        super(SilhouetteVisualizer, self).__init__(
            estimator, ax=ax, is_fitted=is_fitted, **kwargs
        )
```

To appease the linter.